concepts/timeline.md

```diff
@@ -29,6 +38,16 @@ They should be encouraged to be lead in at least 1 project, but this is not requ
 
 Money should start coming in this time, if appreciation of crypto assets didn't extend that period
 
+```{note}
+Can maybe clarify that part, do you expect the project to be mature and revenue-generating after 2 yrs of definition and then 2 yrs of production? What level of revenue generation to expect from the project itself? 
+
+If a project is production-grade and can add value, business dev/marketing/sales can theoretically scale revenue. What is a minimum set of tasks necessary to 'test a market', at least in a very basic form?, and know if more work/money needs to be invested for comercialization / donation-based / consulting based.
+
+I understand you want to avoid going into a full business or financial plan, but developing on the financials can give a more solid basis to the project.
+
+In general, you could also expand the `business model` part to explain what financiation to expect from project, and what from other collective / contributors sources. Are you open to other sources (subventions, donations)? E.g. subvention may require specific forms of collective.
+```
+
```

Comment:
Any type of financing is valid as long as it doesn't clash with the fundamental tenets of the collective.
I understand that some means of financing might require a specific (legal) form of collective, the latter can be changed if the members vote positively on this (meaning that we should choose a legal status at the beginning of the project, but we are open to change it in the future if it proves advantageous)